example_data/sample_bundle.json

```diff
@@ -470,34 +533,77 @@
             ]
           }
         ],
-        "valueString": "Next generation sequence analysis indicates that General Manager is heterozygous for the MECP2 c.953G>C p.(Arg318Pro) likely pathogenic variant that has been confirmed by Sanger sequence analysis"
+        "effectiveDateTime": "2020-10-22",
+        "note": [
+          {
+            "authorString": "genomic interpretation",
+            "text": "Next generation sequence analysis indicates that Maria321 BrighGosh321 is heterozygous for the SMC1A c.1263del p.(Lys422Serfs*28) pathogenic variant that has been confirmed by Sanger sequence analysis (see technical information below)."
+          }
+        ],
```

Comment:
this was previously a valueString, do we have a reason for changing it?

Reply:
Yes, because of the update where we now want to include report findings valueString and valueCodeableConcept results in an error from the server. I then resorted to using note instead valueString